committeeoversightapp/templatetags/committeeoversight_tags.py

```diff
@@ -0,0 +1,32 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter(name='lookup')
+def lookup(value, arg):
+    return value[arg]
+
+
+rating_colors = {
```

Comment:
i'd expect a value that's derived from a model attribute to be defined as a property on the model. what do you say?

Reply:
Interesting! It feels strange to attach a CSS name directly on a model, but I hadn't thought about this option. Would it be a property on the `CommitteeRating` model? The `Organization` model is defined in OCD so I wouldn't want to modify that.

Reply:
yes, on the `ComitteeRating` model. and to be clear, i mean a `@property`, like a method on the model, not like a model attribute that's stored in the database. e.g., https://github.com/datamade/committee-oversight/blob/8ab986b2cee90e83fd925483d8d0727b5ee08d73/committeeoversightapp/models.py#L61-L66

Reply:
Gotcha. Makes sense! I'll switch it over once I'll rebased this from master and have that model.